jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -31,6 +31,14 @@
             description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
     scm:
         - git:
+            url: https://github.com/kubernetes/test-infra
+            basedir: go/src/k8s.io/test-infra
```

Comment:
I think this is going to make the kubernetes builds think the git repo is dirty, since this ends up being a subdirectory of it. (we don't gitignore go/.)